components/org.wso2.carbon.identity.captcha/src/main/java/org/wso2/carbon/identity/captcha/util/CaptchaUtil.java

```diff
@@ -418,6 +433,14 @@ private static void setReCaptchaConfigs(Properties properties) {
         }
         CaptchaDataHolder.getInstance().setReCaptchaRequestWrapUrls(reCaptchaRequestWrapUrls);
 
+        String reCaptchaScoreThreshold = properties.getProperty(CaptchaConstants.RE_CAPTCHA_SCORE_THRESHOLD);
+        try {
+            CaptchaDataHolder.getInstance().setReCaptchaScoreThreshold(Double.parseDouble(reCaptchaScoreThreshold));
+        } catch (NumberFormatException | NullPointerException e) {
+            log.info("Error parsing recaptcha.threshold from config", e);
+            CaptchaDataHolder.getInstance().setReCaptchaScoreThreshold(0.5);
```

Comment:
Don't hardcode the value instead use the config value

Reply:
Shall I remove the value?

Reply:
Added the value to the CaptchaDataHolder class